bitnami/concourse/values.yaml

```diff
@@ -0,0 +1,1144 @@
+  ##
+  extraEnvVarsCM:
+  ## @param web.extraEnvVarsSecret Name of existing Secret containing extra env vars for web nodes
+  ##
+  extraEnvVarsSecret:
+  ## @param web.extraVolumes Optionally specify extra list of additional volumes for the web pod(s)
+  ##
+  extraVolumes: []
+  ## @param web.extraVolumeMounts Optionally specify extra list of additional volumeMounts for the web container(s)
+  ##
+  extraVolumeMounts: []
+  ## @param web.sidecars Add additional sidecar containers to the web pod(s)
+  ## e.g:
+  ## sidecars:
+  ##   - name: your-image-name
+  ##     image: your-image
+  ##     imagePullPolicy: Always
+  ##     ports:
+  ##       - name: portname
+  ##         containerPort: 1234
+  ##
+  sidecars: {}
+  ## @param web.initContainers Add additional init containers to the web pod(s)
+  ## ref: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
+  ## e.g:
+  ## initContainers:
+  ##  - name: your-image-name
+  ##    image: your-image
+  ##    imagePullPolicy: Always
+  ##    command: ['sh', '-c', 'echo "hello world"']
+  ##
+  initContainers: {}
+  
+  ## @param web.existingSecret
+  existingSecret:
+
+
+## @section Worker 
+worker:
+  ## @param worker.enabled
+  enabled: true
+  
+  ## @param worker.replicaCount
+  replicaCount: 2
+  ## @param worker.mode Selects kind of Deployment. Valid Options are: statefulSet | deployment
+  ## Using Deployment leads to ephemeral workers. Meaning workers do not
+  ## share state between restarts
+
+  mode: deployment
+
+  ## @param worker.workDir Directory in which to place container data.
+  workDir: /opt/bitnami/concourse
+
+  ## @param worker.nameOverride provide a name to substitute for the full names of resources
+  nameOverride:
+
+  ## @param worker.logLevel Minimum level of logs to see. Possible options: debug, info, error
+  logLevel: debug
+
+  ## @param worker.keySecretsPath For managing where secrets should be mounted for worker agents
+  keySecretsPath: "/concourse-keys"
+
+  ## @param worker.args 
+  args: ["worker"]
+
+  ## @param worker.lisentIp IP address on which to listen for the Garden server.
+
+  lisentIp: 127.0.0.1
+
+  ## @param worker.containerPort Port on which to listen for the Garden server.
+  containerPort: 7777
+
+  ## Enable HorizontalPodAutoscaler for worker pods
+  ## ref: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+  ##
+  ## @param worker.autoscaling.enabled
+  autoscaling: 
+    enabled: false
+  #      maxReplicas: 1
+  #      minReplicas: 1
+  #      builtInMetrics:
+  #        - type: Resource
+  #          resource:
+  #            name: cpu
+  #            target:
+  #              type: Utilization
+  #              averageUtilization: 80
+  #      customMetrics:
+  #        - type: Pods
+  #          pods:
+  #            metric:
+  #              name: concourse_workers_containers
+  #            target:
+  #              type: AverageValue
+  #              averageValue: 180
+  
```

Comment:
Remove all trailing whitespace, otherwise the lint github action will fail. There are IDE plugins to automatically remove trailing whitespace on file save.